app/src/ui/diff/text-diff.tsx

```diff
@@ -1016,10 +1016,15 @@ export class TextDiff extends React.Component<ITextDiffProps, ITextDiffState> {
 
     const gutterParentElement = cm.getGutterElement()
     const gutterElement = gutterParentElement.getElementsByClassName(
-      'diff-gutter'
+      diffGutterName
```

Comment:
Not related to hotfix - Just a small cleanup as a constant for the magic string `'diff-gutter'` was already defined n the file.